hikari/cli.py

```diff
@@ -43,3 +44,13 @@ def main() -> None:
     sys.stderr.write(f"located at {path}\n")
     sys.stderr.write(f"{py_impl} {py_ver} {py_compiler}\n")
     sys.stderr.write(" ".join(frag.strip() for frag in platform.uname() if frag and frag.strip()) + "\n")
+
+    if extensions := importlib.metadata.entry_points().get("hikari"):
+        sys.stderr.write("\nInstalled extensions:\n")
+
+        for extension in extensions:
+            module = extension.load()
+            module_path = os.path.abspath(os.path.dirname(module.__file__))
+            version = getattr(module, "__version__", "unknown version")
```

Comment:
have you considered trying `importlib.metadata.version` here?

Reply:
Initially did, but had a dumb mistake and couldnt get it work. Now i realized what that dumb mistake was and switched to it

Thanks for pointing it out and having me have another look at it!